auth-web/src/util/constants.ts

```diff
@@ -247,6 +247,48 @@ export enum NrEntityType {
     INFO = 'INFO', // special value for sub-menu
 }
 
+export enum NrRequestTypes {
+  // Name Change
+  BEC = 'Name Change',
+  CCC = 'Name Change',
+  CCP = 'Name Change',
+  CCR = 'Name Change',
+  CFR = 'Name Change',
+  CUL = 'Name Change',
+
+  // Continuation in
+  BECT = 'Continuation In',
+  CCCT = 'Continuation In',
+  CTC = 'Continuation In',
+  CT = 'Continuation In',
+  ULCT = 'Continuation In',
+
+  // Alteration
+  BECV = 'Alteration',
+  CCV = 'Alteration',
+  BECR = 'Alteration',
+  UC = 'Alteration',
```

Comment:
1. check the other enum to identify which codes are conversions
2. handle them according to what we are discussing in Standup

Do you know if there are NR request  types not in this enum?

Reply:
Looking at the [namex](https://github.com/bcgov/namex/blob/7e5c88a86a0585a404bafc549a8755b74264b06a/api/namex/services/lookup/name_request_filing_actions.py#L51) file there are some that are not in this enum. The ones I saw are: Assumed Name and Reinstatement.